### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by KaskMIL**
# In this new Pull Request, we are adding more dynamics with a Pop-Up Window.
- In the first instance we made a script to create the different cards in the **Works Section** dynamically.
- Then we add the **structure**, **styles** and **script** to the new **Pop-Up window**, so every time you press on the "See Project" button, the Pop-Up window appears and prints a blur all over the body beneath.

### CODE REVIEW

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi @KaskMIL, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that has linter errors. Kindly fix the linter errors before submitting them for review. I have messaged on slack about how to fix linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: dist/css/main.css [hunk 1]
```diff
@@ -979,4 +983,152 @@ footer ul li a img {
             transform: scale(1.5);
   }
 }
+
+.popup {
```
Comment (Ismailco) [inline dist/css/main.css:987]: - [ ] The popup needs to follow the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colors, images, margins), kindly change the close button ```color```, ```font-weight``` and add some ```padding``` to the buttons so your design be as close as possible to the Figma design

- Your design:
Screenshot_20220511_222312

- Figma design:
Screenshot_20220511_222329

File: script.js [hunk 2]
```diff
@@ -38,4 +47,149 @@ $aboutNavMobile.addEventListener('click', () => {
 
 $contactNavMobile.addEventListener('click', () => {
   hideMobileMenu();
-});
\ No newline at end of file
+});
+
+const $worksSection = document.getElementById('works-section');
+
+const ProjectList = [
```
Comment (Ismailco) [inline script.js:54]: - [ ] Good job implementing the array of object to store project info however each object need to contain at least the following information about the project: ```name```, ```description```, ``` featured image```, ```technologies```, ```link to live version```, and ```link to the project source code```, kindly add the link to the project source code and project live version

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @KaskMIL, @BregornOriginal,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KaskMIL** [comment]
Hi @Ismailco,
Thanks for your review, I think I could implement all the changes.
thanks agains

File: script.js (hunk 2 above)
Comment (sja-thedude) [reply script.js:54]: Good job implementing the changes 👍

**Comment by KaskMIL** [comment]
Hi @sja-thedude,
Thanks for your review!
Have a good one!

**Pull request merged**